frontend/packages/dev-console/src/components/import/git/SourceSecretSelector.tsx

```diff
@@ -1,46 +1,91 @@
+/* eslint-disable no-unused-vars, no-undef */
 import * as React from 'react';
-import { FormGroup, ControlLabel, FormControl, HelpBlock } from 'patternfly-react';
+import { FormGroup, ControlLabel } from 'patternfly-react';
+import { useFormikContext, FormikValues, useField } from 'formik';
+import { createModalLauncher } from '../../../../../../public/components/factory/modal';
 import SourceSecretDropdown from '../../dropdown/SourceSecretDropdown';
-import CreateSourceSecret, { SecretType } from './CreateSourceSecret';
+import {
+  withSecretForm,
+  SourceSecretForm,
+  SecretTypeAbstraction,
+} from '../../../../../../public/components/secrets/create-secret';
```

Comment:
Reference code in the `public` dir using `@console/internal`

Reply:
Done https://github.com/openshift/console/commit/462df22ebef1682c8ec550a84bf3d529bc276c87